jdaviz/cli.py

```diff
@@ -85,15 +88,17 @@ def main(filename, layout='default', instrument=None, browser='default',
         os.chdir(start_dir)
 
 
-def _main():
+def _main(config=None):
```

Comment:
This change seems unrelated. What is the motivation for this?

Reply:
Each one of the config launchers call `_main` with their `config` specified. Notice how they're calling `_main` and not `main` (no underline) to handle the argument parsing for them